### DISCUSSION TITLE
Флаги для запуска приложения сразу максимизированным или фуллскрин

### DISCUSSION THREAD

**Discussion by mopo3ilo** [cucumber-sp/yandex-music-linux#116] [Ideas]
Собственно сабж. Хотелось бы иметь флаги запуска (лучше в конфиге), чтобы запускался сразу развёрнутым.
А если добавить флаги на выполнение скриптов на автостарт волны и развороту обложки, было б счастье.

**Comment by cucumber-sp** [comment]
С обновлением версии электрон теперь запускается на весь экран. Если вдруг захотите сделать параметром - на то и опен сорс :)

**Comment by tycoon3-notcoder** [comment]
is there a flag to start so that music can be played immediately?
